docs/index.md

```diff
@@ -59,7 +59,7 @@ steampipe plugin install googleworkspace
 | :---------- | :-----------|
 | Credentials | 1. To use **domain-wide delegation**, generate your [service account and credentials](https://developers.google.com/admin-sdk/directory/v1/guides/delegation#create_the_service_account_and_credentials) and [delegate domain-wide authority to your service account](https://developers.google.com/admin-sdk/directory/v1/guides/delegation#delegate_domain-wide_authority_to_your_service_account). Enter the following OAuth 2.0 scopes for the services that the service account can access:<br />`https://www.googleapis.com/auth/calendar.readonly`,<br />`https://www.googleapis.com/auth/contacts.readonly`,<br />`https://www.googleapis.com/auth/contacts.other.readonly`,<br />`https://www.googleapis.com/auth/directory.readonly`,<br />`https://www.googleapis.com/auth/drive.readonly`,<br />`https://www.googleapis.com/auth/gmail.readonly`<br />2. To use **OAuth client**, configure your [credentials](#authenticate-using-oauth-client). |
 | Radius      | Each connection represents a single Google Workspace account. |
-| Resolution  | 1. Credentials from the JSON file specified by the `credential_file` parameter in your Steampipe config.<br />2. Credentials from the JSON file specified by the `token_path` parameter in your Steampipe config.<br />3. Credentials from the default json file location (~/.config/gcloud/application_default_credentials.json). |
+| Resolution  | 1. Credentials from the JSON file specified by the `token_path` parameter in your Steampipe config.<br />2. Credentials from the default json file location (`~/.config/gcloud/application_default_credentials.json`).<br />3. Credentials from the JSON file specified by the `credentials` parameter in your Steampipe config. |
```

Comment:
Is this credential resolution correct? In `service.go`, it seems like we're checking if the credential content is set first, and if so use that; else, check if token path is set, and then use that if available. I'm not sure how exactly we use the default ADC creds, but I'm assuming if we don't return any session config and attempt to create a new service, default ADC creds are used?

This question came up for me in the Google Directory plugin changes as well (https://github.com/turbot/steampipe-plugin-googledirectory/pull/32), so I checked back in the Google Sheets plugin and saw we had the following resolution order listed:
```
1. Credentials from the JSON file specified by the token_path parameter in your Steampipe config.
2. Credentials from the default json file location (~/.config/gcloud/application_default_credentials.json).
3. Credentials from the JSON file specified by the credentials parameter in your Steampipe config.
```

But when I looked in the code (https://github.com/turbot/steampipe-plugin-googlesheets/blob/main/googlesheets/service.go#L151-L167), it looked like the order was actually:
```
1. Credentials from the JSON file specified by the token_path parameter in your Steampipe config.
2. Credentials from the JSON file specified by the credentials parameter in your Steampipe config.
3. Credentials from the default json file location (~/.config/gcloud/application_default_credentials.json).
```

So overall, I think we should discuss:
- What is the credential resolution order meant to be? What does the [Google API Go Client](https://github.com/googleapis/google-api-go-client) we're using use in terms of creds precedence?
- Does the Google Sheets plugin follow this order correctly?
- Are there any updates we need to make in this PR, and in https://github.com/turbot/steampipe-plugin-googledirectory/pull/32 as well?